tests/conftest.py

```diff
@@ -397,3 +401,6 @@ def _subprocess(cmd):
         for x in p:
             x.terminate()
             x.wait()
+
```

Comment:
Global is unnecessary

Reply:
That was my original approach, but a global does appear necessary. When `supervisor_uri` gets set - first appearing here:
https://github.com/radiasoft/sirepo/blob/e80ddebeaed4493e940a0f493f61352a311354fe/sirepo/job_api.py#L353-L355
and used here:
https://github.com/radiasoft/sirepo/blob/e80ddebeaed4493e940a0f493f61352a311354fe/sirepo/job_api.py#L310-L311
it does not change until the next test file runs, while `_subprocess_setup` runs for every test case. That causes connection errors like this:
```
Sep 27 00:48:51 285570     0 ../../sirepo/pkcli/job_supervisor.py:78:default_command ip=127.0.0.1 port=10368
sirepo/job_api.py:231:request api=api_beginSession runDir=None
sirepo/session.py:81:_begin error=HTTPConnectionPool(host='127.0.0.1', port=10891): Max retries exceeded with url: /job-api-request (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fdf113c1fd0>: Failed to establish a new connection: [Errno 111] Connection refused')) trying job_api.begin_session stack=Traceback (most recent call last):
```
You can see the two different ports (which matched for the 1st test). Is the global `_PORT` working around a "bug" then?

Reply:
ping for feedback on https://github.com/radiasoft/sirepo/pull/4846#discussion_r982451782

Reply:
It is a bug. It might be that _auth_fc_module needs a `fixture(scope="module")`. I don't know which tests are failing, but once the supervisor starts, it shouldn't be started again. The config should be fixed for the whole module.

Reply:
Adding the fixture scope did the trick here, but now supervisor_purge_free_sims_test.py fails on the 2nd test case (the status is "completed" instead of "job_run_purged") and I haven't figured that out yet. The same happens if I add the scope to a fresh master checkout with no other changes. I'll keep at it...

Reply:
Update: the module-scoped _`auth_fc_module` version sees the purge loop collect and purge the json file for the 2nd test while it is still running, while the unscoped version skips it.

Reply:
Update: the problem was the the "adjustTime" call - previously, when a new supervisor was created for every test case, the time shift was freshly applied. With a single supervisor, the time shift gets applied twice, pushing subsequent sim files outside the purge threshold.